src/pages/TestingModelPage.tsx

```diff
@@ -25,6 +45,30 @@ const TestingModelPage = () => {
     }
   }, [model, navigateToDataSamples]);
 
+  const { openEditor, resetProject, projectEdited } = useProject();
```

Comment:
I moved this from the grid/table component to match the arrangement for the data samples page.
The difference between the two had often confused me.